api/api-rule-violations-known.list

```diff
@@ -152,6 +153,7 @@ API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,KubeVirtConfi
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,KubeVirtConfiguration,SupportedGuestAgentVersions
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,KubeVirtList,Items
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,KubeVirtStatus,Conditions
+API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,NodePlacement,Tolerations
```

Comment:
should we add the list type annotation for this and Filesystems instead of adding them to the known violations list?